### ISSUE TITLE
Support matchers for `equals` or `matchesElement`

### ISSUE THREAD

**Issue by wdoug** [blainekasten/enzyme-matchers#37]
For enzyme [.equals](http://airbnb.io/enzyme/docs/api/ShallowWrapper/equals.html), it would be great to be able to write:

``` jsx
const wrapper = shallow(<span />);
expect(wrapper).toEqualNode(<span />);
```

instead of 

``` jsx
const wrapper = shallow(<span />);
expect(highlightedText.equals(<span />)).toBe(true);
```

which gives no useful information on failure.

Or the equivalent with [.matchesElement](http://airbnb.io/enzyme/docs/api/ShallowWrapper/matchesElement.html)

**Comment by blainekasten** [comment]
That does sound like a good addition. I'm currently on paternity leave for
another month. If you want to put up a PR though I can try and get it
merged sooner than that!

**Comment by cwmoo740** [comment]
https://github.com/blainekasten/jasmine-enzyme/pull/43

**Comment by blainekasten** [comment]
Going to need a new PR. That one was closed and the project was reworked a lot.

**Comment by blainekasten** [comment]
@cwmoo740 @wdoug would either of you want to implement this?

**Comment by blainekasten** [comment]
closed by #95
